var/spack/repos/builtin/packages/atlas/package.py

```diff
@@ -55,10 +55,15 @@ class Atlas(Package):
 
     variant('shared', default=True, description='Builds shared library')
     variant('pthread', default=False, description='Use multithreaded libraries')
+    variant('ilp64', default=False, description='64 bit integers.'
+            'This option is currently not supported by Atlas and it is here'
+            'only for consistency with other BLAS/LAPACK providers')
 
     provides('blas')
     provides('lapack')
 
+    conflicts('+ilp64', when='@0.1:')
```

Comment:
I don't know how I feel about this... I understand that you want things to be uniform, but I am not sure you can say:
```python
depends_on('lapack+ilp64')
```
and in that case I think it should be a property of the virtual dependency, not the package that provides it. @adamjstewart @tgamblin Opinions on this?

Reply:
sure, i agree. I just though that this could be a one step closer to https://github.com/LLNL/spack/issues/1712